Update Twilio.markdown (#30122)
home-assistant/home-assistant.github.io: source/_integrations/twilio.markdown

## Configuration

> To use this notification integration in your installation, you must first install the base integration from Settings _after_ adding your Twilio credentials to your `configuration.yaml` file (either directly or using [secrets](https://www.home-assistant.io/docs/configuration/secrets/)):

Reviewer: I see the confusion. We need to insert the configuration flow snippet the provides the instructions on how to set it up from the UI (which is currently missing).

Also, the reference to secrets isn't relevant here and should be removed.

Author: New to Gihub, and it's kicking my ass.  How about:  "1) add your Twilio credentials to `configuration.yaml`, and then 2) install the base integration".  I see your point about secrets.  But documentation is for new users and not expereienced ones.  I'm new to HA, didn't yet know about HA secrets, lost a bit of time trying to figure it out.  I was hoping to leave a breadcrumb for the next "secrets unfamiliar" user.

### Usage

> After adding your Twilio account credentials to `configuration.yaml` and the Twilio base integration from Settings, you should restart Home Assistant.   Then, you can configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.

Reviewer: There is no restart needed after setting things up from the UI. There is, however, a restart needed when changing YAML configuration. There is an order issue in your description.

```suggestion
After adding your Twilio account credentials to `configuration.yaml`, restart Home Assistant. Next, add the Twilio base integration. Then, you can configure either or both of the [Twilio SMS](/integrations/twilio_sms) and [Twilio Phone](/integrations/twilio_call) integrations to utilize the notification functionality.
```
